src/Google/RequestReviewStatuses.php

```diff
@@ -78,9 +80,31 @@ public function get_statuses_from_response( array $response ) {
 
 		return [
 			'issues'   => array_map( 'strtolower', array_unique( $issues ) ),
-			'cooldown' => $cooldown * 1000,
+			'cooldown' => self::parse_cooldown( $cooldown ), // add lifetime cache to cooldown time
 			'status'   => $status,
 		];
 	}
 
+	/**
+	 * Allows a hook to modify the lifetime of the Account review data.
+	 *
+	 * @return int
+	 */
+	public function get_account_review_lifetime(): int {
+		return apply_filters( 'woocommerce_gla_mc_account_review_lifetime', self::MC_ACCOUNT_REVIEW_LIFETIME );
+	}
+
+	/**
+	 * @param int $cooldown The cooldown in PHP format (seconds)
+	 *
+	 * @return int The cooldown in miliseconds and adding the lifetime cache
+	 */
+	private function parse_cooldown( $cooldown ) {
```

Comment:
Probably left over from a refactor, but it seems the `$cooldown` is already passed through `intval( strtotime() )` before calling the function, so it's always passed to this function as an int.
So we can force the function parameter to int and probably rename the function to `calculate` or `get` instead of `parse`.